assets/javascript/app.js

```diff
@@ -1,164 +1,10 @@
-            });
-    };
-
-    var displayFoodReportResults = function (results, foodNumber, buttonParentDiv) {
-        console.log("here");
-        // var displayDiv = $('#food-report-results');
-        var reportWrapTable = $("<table class='report-wrapper'></table>");
-        reportWrapTable.attr("id", foodNumber);
-
-        var foodId = results.foods[0].food.desc.ndbno;
-        var foodIdRow = $('<tr>');
-        foodIdRow.html("<td class='data-heading-col'>Title:</td>" + "<td class='data-col'>" + foodId + "</td></tr>");
-        reportWrapTable.append(foodIdRow);
-        
-        var foodName = results.foods[0].food.ing.desc;
-        var foodNameRow = $('<tr>');
-        foodNameRow.html("<td class='data-heading-col'>Name:</td>" + "<td class='data-col'>" + foodName + "</td></tr></table>");        
-        reportWrapTable.append(foodNameRow);
-
-        var nutrientsRows = function () { 
-            console.log("hi loop", results.foods[0].food.nutrients.length);  
-            console.log("hi name: ", results.foods[0].food.nutrients[0].name);         
-            for (var i = 0; i < results.foods[0].food.nutrients.length; i++) {
-                console.log("loop#: ", i);
-                var title = results.foods[0].food.nutrients[i].name;
-                var value = results.foods[0].food.nutrients[i].value;
-                var nutrientRow = $('<tr>');
-                nutrientRow.html("<td class='data-heading-col'>" + title + ":</td>" + "<td class='data-col'>" + value + "</td></tr>");
-                reportWrapTable.append(nutrientRow);
-            }
-        }
-        nutrientsRows();
-        buttonParentDiv.append(reportWrapTable);
-    };
-
-/** AJAX request to the Giphy picturesport
-var foodReportAjax = function (foodNumber, buttonParentDiv) {
-    if (!foodNumber) {
-        console.log("false");
-        return false;
-    }
-    // https://api.nal.usda.gov/ndb/V2/reports?ndbno=01009&ndbno=01009&ndbno=45202763&ndbno=35193&type=b&format=json&api_key=DEMO_KEY
-    var APIKEY = "MTThsOXeyC4yDoAe048samFSx66c0bbwi0HO6m4G";
-    var queryURL = "https://api.nal.usda.gov/ndb/V2/reports?ndbno=" + foodNumber + "&format=json&q=" + "&max=5&offset=0" + "&api_key=" + APIKEY;
-    console.log("foodReportAjax queryURL: ", queryURL);
-    // AJAX request with the queryURL
-    $.ajax({
-        url: queryURL,
-        method: "GET"
-    })
-        .then(function (response) {
-            // storing the data from the AJAX request in the results variable
-            var results = response;
-            console.log("foodReportAjax results:", results);
-            displayFoodReportResults(results, foodNumber, buttonParentDiv);
-        });
-}; */
-
-
-    // Initialize function
-    var init = function () {
-        console.log("---- hi init ----");
-        $('.jumbowrap').show();
-        foodSearchAjax();
-        // here will be the picture search ajax giphy rated G
-
```

Comment:
function displayGifs(input) {
        var anim = input;
        var queryURL = "https://api.giphy.com/v1/gifs/search?q=" + anim + "&api_key=rozTdp9Yz335rrxK5i05ehFZQ00hDBP0&limit=3&rating=y";
    
        $.ajax({
            url: queryURL,
            method: 'GET'
        })
    
        .done(function(response) {
            
            var results = response.data;


          for (var i = 0; i < results.length; i++) {

 
            var animalDiv = $("<div>");

            var p = $("<p>").text("");


            var animalImage = $("<img>");
     
            animalImage.attr("src", results[i].images.fixed_height_small.url);
                if(i===0){
                    $("#foodGif1").prepend(animalDiv);
                }
                else if(i===1){
                    $("#foodGif2").prepend(animalDiv);
                }
                else if(i===2){
                    $("#foodGif3").prepend(animalDiv);
                }
  
            animalDiv.append(p);
            animalDiv.append(animalImage);

          }
        });
    }